.github/workflows/main.yml

```diff
@@ -246,3 +246,10 @@ jobs:
         cd target
         curl -sSLO https://raw.githubusercontent.com/rust-lang/rust/master/src/tools/linkchecker/linkcheck.sh
         sh linkcheck.sh --all --path ../src/doc cargo
+
+  credential_msrv:
```

Comment:
Should we make it a required job?

(I am brain-fogged now sorry if this doesn't make sense)

Reply:
Yes, thanks!